nf_core/pipeline-template/nextflow.config

```diff
@@ -86,95 +86,102 @@ try {
-        apptainer.enabled      = false
+        singularity.enabled     = true
+        singularity.autoMounts  = true
+        conda.enabled           = false
+        docker.enabled          = false
+        podman.enabled          = false
+        shifter.enabled         = false
+        charliecloud.enabled    = false
+        apptainer.enabled       = false
     }
     podman {
-        podman.enabled         = true
-        conda.enabled          = false
-        docker.enabled         = false
-        singularity.enabled    = false
-        shifter.enabled        = false
-        charliecloud.enabled   = false
-        apptainer.enabled      = false
+        podman.enabled          = true
+        conda.enabled           = false
+        docker.enabled          = false
+        singularity.enabled     = false
+        shifter.enabled         = false
+        charliecloud.enabled    = false
+        apptainer.enabled       = false
     }
     shifter {
-        shifter.enabled        = true
-        conda.enabled          = false
-        docker.enabled         = false
-        singularity.enabled    = false
-        podman.enabled         = false
-        charliecloud.enabled   = false
-        apptainer.enabled      = false
+        shifter.enabled         = true
+        conda.enabled           = false
+        docker.enabled          = false
+        singularity.enabled     = false
+        podman.enabled          = false
+        charliecloud.enabled    = false
+        apptainer.enabled       = false
     }
     charliecloud {
-        charliecloud.enabled   = true
-        conda.enabled          = false
-        docker.enabled         = false
-        singularity.enabled    = false
-        podman.enabled         = false
-        shifter.enabled        = false
-        apptainer.enabled      = false
+        charliecloud.enabled    = true
+        conda.enabled           = false
+        docker.enabled          = false
+        singularity.enabled     = false
+        podman.enabled          = false
+        shifter.enabled         = false
+        apptainer.enabled       = false
     }
     apptainer {
-        apptainer.enabled      = true
-        apptainer.autoMounts   = true
-        conda.enabled          = false
-        docker.enabled         = false
-        singularity.enabled    = false
-        podman.enabled         = false
-        shifter.enabled        = false
-        charliecloud.enabled   = false
+        apptainer.enabled       = true
+        apptainer.autoMounts    = true
+        conda.enabled           = false
+        docker.enabled          = false
+        singularity.enabled     = false
+        podman.enabled          = false
+        shifter.enabled         = false
+        charliecloud.enabled    = false
+    }
+    wave {
+        apptainer.ociAutoPull   = true
+        singularity.ociAutoPull = true
+        wave.enabled            = true
+        wave.freeze             = true
```

Comment:
I see in the documentation that we would need to provide `wave.build.repository` together with `wave.freeze = true`. Is it required? Would it be safer to let users provide both parameters.